packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts

```diff
@@ -964,7 +964,7 @@ describe('StakePoolHttpService', () => {
             });
             it('pledgeMet false, status activating, and condition', async () => {
```

Comment:
Not sure the original intent of the test, but that's what it's doing now
```suggestion
            it('pledgeMet false, status active, and condition', async () => {
```

Reply:
It's hard to say which was the original intent of the test... it's inside a `identifier & status & pledgeMet filters` describe block, so I guess the intent of the test is to check that logical expression between filters work as expected.
Anyway the status of a stake pool in the fixtures is strongly dependent on the time when the snapshot was taken... probably recent changes introduced make no pools to have the right combination to match the filter in this tests so I had only two options: disabling the test or change it as I did.
In my opinion, tests about stake pool status should be done only as long running tests which prepare the pools as per their needs, so we can be sure about the time when they run.
Until that improvement is done... 94e3d9438a should solve this